### DISCUSSION TITLE
Como organizar um Evento || Meetup ?

### DISCUSSION THREAD

**Discussion by Cleysonlb** [frontendbr/forum#209] [Discussão]
Fala galera beleza?! Gostaria de abordar esse assunto, pois moro em uma cidade pequena e tem uma galera aqui que carece de meetups / eventos e ficar indo para grandes centros sempre é complicado, então estou com umas ideias de juntar uns amigos e organizar algo pra ver no que da!

**Comment by felipefialho** [comment]
@raphaelfabeni, ainda tem aquele guia que escrevemos?

**Comment by notnotgabriel** [comment]
Em Campinas nós criamos um grupo na plataforma do [meetup.com](meetup.com/pt-BR/Campinas-Front-End-Meetup/), e começamos a divulgar para os amigos mais próximos e no facebook.
Logo no começo já marcamos um meetup com 2 palestras e a galera compareceu em peso :grin: 
@Cleysonlb de qual cidade você é?

**Comment by Cleysonlb** [comment]
@gabriel-ribeiro-ir sou de Teresópolis região serrana do RJ, aqui estamos cercados de cidades pequenas, muitas são perto, criando um meetup podemos juntar até galera dessas cidades!

**Comment by woliveiras** [comment]
Tem o esquema do FEMUG que é legal também ein: https://github.com/femug/femug

**Comment by barbier** [comment]
Por favor, me dêem (mais) uma desculpa para subir a serra! :D

**Comment by thiamsantos** [comment]
@woliveiras muito legal esse projeto do femug!!!

A parte que eu mais gostei foi essa:

> Caso seja criado um grupo para todo o estado, o mesmo deverá ter como sede a capital do mesmo. Por exemplo: "FEMUG-RS" deverá ter como base, Porto Alegre. Para "FEMUG-MG", a base seria Belo Horizonte e assim por diante. Para um "FEMUG-ACRE", o servidor deverá retornar um erro HTTP 404.

**Comment by Cleysonlb** [comment]
@barbier aqui tem a cerveja therezopolis gold, é um ótimo motivo pra subir a serra!!

**Comment by raphaelfabeni** [comment]
Fala @Cleysonlb ..
Cara, eu e o @LFeh apesar de estarmos em SP que é uma cidade enorme, ficamos muito receosos em como começar o [meetup CSS](https://github.com/raphaelfabeni/css-sp) mas resolvemos tentar. [Escrevemos um pouco aqui](https://github.com/raphaelfabeni/css-sp/blob/master/inicio.md)  e eu [escrevi também um post no meu blog](http://www.raphaelfabeni.com.br/nascimento-meetup-css/).
Acho que é isso. Valeu!

**Comment by Cleysonlb** [comment]
@raphaelfabeni vlw por dividir essa experiência!! Vou estudar direitinho as possibilidades, se rolar eu dou o feedback aqui no forum ;)

**Comment by barbier** [comment]
@Cleysonlb e se não rolar, dê um feedback também, por favor!
Acho importante que tanto as experiências boas ou ruins sejam compartilhadas.
Edit.: Se eu subir dirigindo não poderei beber, mas acompanho vocês! =)

**Comment by Cleysonlb** [comment]
@barbier pode deixa! Caso não role eu posto aqui as dificuldades que encontrei e ou o pq de não ter rolado!!

**Comment by devigor** [comment]
Interessante, também sou de Teresópolis e sinto falta de eventos aqui. Seria bem massa organizar um meetup aqui.

**Comment by danilovaz** [comment]
@Cleysonlb Eu via esse mesmo problema onde moro, em São José dos Campos / SP. Apesar de ser relativamente perto de SP, é complicado ir pra sampa no dia de semana e voltar pra trampar no outro dia. E aqui ainda rolava até que bastante eventos anuais de programação em geral (java, python, ruby, js), mas nada de Meetups mensais. Sentia a necessidade de agitar um pouco as coisas na cidade e fomentar a comunidade local.

Daí me juntei com outros caras e minas que tinham a mesma percepção que eu e em 2016 surgiu aqui o BeerJS e um braço do FrontUX. E vamos tocar os 2 esse ano, porque o feedback da galera foi muito bom. 

Nosso saldo no ano passado foi: 

- BeerJS-SJC: 183 membros, 5 Meetups realizados e 96 inscritos em todos os Meetups;
- FrontUX-SJC: 113 membros, 2 Meetups realizados e 37 inscritos em todos os Meetups.

Pra nossa região, são números bons e que vem crescendo. Além de agitar a comunidade local, nos preocupamos em ser uma comunidade aberta para todos. Sem exceções. Na organização do BeerJS, temos mulheres que estão fortalecendo a comunidade local, tivemos até um Meetup onde a maioria era mulher. Ambiente amigável e muita troca de ideia.

Todo esse pano de fundo, pra te falar que o que (eu acho) você precisa pra organizar um Meetup é:
- Ter um tempo pra se dedicar em organizar dia, formato, vagas;
- Organizar os temas e direcionar o evento no dia, pra não virar falatório e garantir que todos os participantes sejam respeitados;
- Usar uma plataforma pra não se perder, como a plataforma do Meetup;
- E ter paciência, pode ser que nos primeiros Meetups aparecerão poucas pessoas, não desiste. Persiste que vai aumentando aos poucos, depende disso: frequência, qualidade da conversa e respeito.

Espero que ajude :smile:
